pkg/project/project.go

```diff
@@ -1,7 +1,7 @@
 package project
 
 var (
-	bundleVersion = "2.0.0-dev"
+	bundleVersion = "2.0.0-rc1"
```

Comment:
Shouldn't the version stay `-dev`?

Reply:
Yes I think so but the diff was weird so I tried to put it back to the previous value. Anyway in master it is back to `-dev`.